src/database/Models/hotel.js

```diff
@@ -13,12 +13,19 @@ module.exports = (sequelize, DataTypes) => {
       // define association here
     }
   };
-  sampleModel.init({
-    name: DataTypes.STRING,
-    tel: DataTypes.STRING
+  hotel.init({
+    hotelName: DataTypes.STRING,
+    priceRange: DataTypes.STRING,
+    location: DataTypes.STRING,
+    ranking: DataTypes.STRING,
+    parking: DataTypes.STRING,
+    wifi: DataTypes.STRING,
+    swimmingPool: DataTypes.STRING,
+    breakfast: DataTypes.STRING,
+    rooms: DataTypes.ARRAY(DataTypes.STRING)
```

Comment:
I think you need also to add hotel avatar since we will need image of the hotel to be displayed on the front-end.

Reply:
Resolved!